app/indexers/hyrax/file_set_indexer.rb

```diff
@@ -1,57 +1,115 @@
 # frozen_string_literal: true
+
 module Hyrax
-  class FileSetIndexer < ActiveFedora::IndexingService
-    include Hyrax::IndexesThumbnails
-    include Hyrax::IndexesBasicMetadata
-    STORED_LONG = ActiveFedora::Indexing::Descriptor.new(:long, :stored)
-
-    def generate_solr_document # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
-      super.tap do |solr_doc|
-        solr_doc['hasRelatedMediaFragment_ssim'] = object.representative_id
-        solr_doc['hasRelatedImage_ssim'] = object.thumbnail_id
+  ##
+  # Indexes Hyrax::FileSet objects
+  class FileSetIndexer < Hyrax::ValkyrieIndexer
+    include Hyrax::ResourceIndexer
```

Comment:
This is the refactor of `Hyrax::FileSetIndexer` to be the indexer for `Hyrax::FileSet` (valkyrie resource) instead of `::FileSet` (AF object).  This is needed because `Hyrax::ValkyrieIndexer #for` method uses naming convention to determine the indexer to use for a given Valkyrie resource.